doc/modules/mod_mam.md

```diff
@@ -25,14 +25,25 @@ For now `odbc` backend has very limited support for this feature, while `cassand
 ### Options
 
 * **backend** (atom, default: `odbc`) - Database backend to use. `odbc`, `riak` and `cassandra` are supported.
-* **add_archived_element** (atom, default: `false`) - Add `<archived/>` element from MAM v0.2.
+* **add_archived_element** (boolean, default: `false`) - Add `<archived/>` element from MAM v0.2.
 * **is_archivable_message** (module, default: `mod_mam_utils`) - Name of a module implementing [`is_archivable_message/3` callback](#is_archivable_message) that determines if the message should be archived.
-* **host** (string, default: `"conference.@HOST@"`) - MUC host that will be archived if MUC archiving is enabled. 
  **Warning**: if you are using MUC Light, make sure this option is set to the MUC Light domain.
 * **pm** (list | `false`, default: `[]`) - Override options for archivization of one-to-one messages. If the value of this option is `false`, one-to-one message archive is disabled.
 * **muc** (list | `false`, default: `false`) - Override options for archivization of group chat messages. If the value of this option is `false`, group chat message archive is disabled.
 
-All options described in this document can be overriden for a specific type of messages through `pm` and `muc` options, e.g.:
+**backend**, **add_archived_element** and **is_archivable_message** will be applied to both `pm` and `muc` (if they are enabled), unless overriden explicitly (see example below).
```

Comment:
They can still be given via top-level option instead of creating a more complicated configuration where `pm` and `muc` are given their own arguments.

e.g. this still works:
```erlang
{mod_mam_meta, [{archive_groupchats, true}]}.
```

and there's no need to write this:
```erlang
{mod_mam_meta, [{pm, [{archive_groupchats, true}]}]}.
```